### DISCUSSION TITLE
2024년 08월 30일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#107] [General]
# 2024년 08월 30일

LOG 오늘은 친절한 독일 이웃 Max 와 Grace 를 만났다. 친절한 Googler 이웃을 어떻게 만나게 되었는지 나도 의문이다… 저녁으로 터키 음식을 먹으러 갔는데 심지어 사주셨다!!! 다음에 꼭 집에서 저녁을 대접해야겠다고 생각했다. 만나러 갈 때는 무슨 얘기할 지 걱정했는데 이런저런 이야기하면서 되게 재미있게 대화하다 왔던 거 같다. 가는 길에 …

https://vero.wiki/2024-08-30/

**Comment by carsago** [comment]
아 여기 좀 오랜만이네
